app/views/layouts/admin.html.erb

```diff
@@ -0,0 +1,39 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <!-- Bootstrap CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+
+    <title>Employee Recognition Platform<</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag 'application', media: 'all' %>
+    <%= javascript_pack_tag 'application' %>
+  </head>
+  <body>
+  <%= render 'pages/adminheader' %>
+  <br/>
+  <div class="container">
+    <% if notice %>
+      <%= render 'pages/notice' %>
+    <% end %>
+    <%= yield %>
+  </div>
+    <!-- Optional JavaScript; choose one of the two! -->
```

Comment:
Leaving comments is not a good practice :)

Reply:
I deleted the comments